All_Relationships/2.candidate-labeler.py

```diff
@@ -146,7 +140,27 @@
 
 
 from  sqlalchemy.sql.expression import func
-labeler = LabelAnnotator(lfs=list(LFS.values()))
+labeler = LabelAnnotator(lfs=list(CG_LFS["CG_DB"].values()) + 
+                         list(CG_LFS["CG_TEXT"].values()) +  
+                         list(CG_LFS["CG_BICLUSTER"]) + 
+                         list(DG_LFS["DG_TEXT"].values()))
+
+
+# In[ ]:
+
+
+def make_sentence_df(candidates):
```

Comment:
I would suggest adding comments for your different functions so that when you return to these scripts later you remember what they were input/output is and what they were used for

Reply:
Same for get_split that is above